hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarboniteWorkerActor.scala

```diff
@@ -29,11 +29,12 @@ class CarboniteWorkerActor(carboniterConfig: HybridCarboniteConfig,
   val carboniteFreezerActor = context.actorOf(CarbonitingMetadataFreezerActor.props(carboniterConfig, self, serviceRegistryActor, ioActor))
 
   val backOff = {
-    val scanConfig = carboniterConfig.freezeScanConfig
+    val freezingConfig = carboniterConfig.freezingConfig
     SimpleExponentialBackoff(
-      initialInterval = scanConfig.initialInterval,
-      maxInterval = scanConfig.maxInterval,
-      multiplier = scanConfig.multiplier,
+      // This actor is only created if the initial interval is finite.
+      initialInterval = freezingConfig.initialInterval.asInstanceOf[FiniteDuration],
```

Comment:
This assumption makes me a little nervous. 

If this is always true, can we bake the knowledge into the type system? Could we pass it in as a parameter? Or make the `carboniterConfig` a validated subtype of `HybridCarboniteConfig` (`FreezeEnabledHybridCarboniteConfig`, maybe?)